### PR TITLE
Wael Miller hw01 assignment

### PR DESCRIPTION
**Pull request by waeltmiller**
**Reporting the process**

Strictly speaking, the task was easy. Nonetheless, it took a great deal of time. Entering a new coding environment ab initio always necessitates referring back to the textbook twice a minute, but in this case the textbook in question was constituted by whatever I could pull up online. 

Helpful tutorials: the syllabus, and contained links

What was easy: most of the tasks

What was hard: working through the deletion issue we had on Friday

Problems solved: I am reminded of something I heard from Professor Stefano Allesina here at the university. To paraphrase: any problem can be answered, but only if the lexical information you have on that problem is sufficiently precise for Google to pull up a solution. If you can't google it, you can't solve it. Accordingly, I see more clearly now that the terminology of coding is perhaps just as important to understand as are the syntax and commands themselves.

### CODE REVIEW

**Review by YixiC94 — COMMENTED** [review]
Very detailed review of the process and challenges encountered when doing HW1. 
Impressive story-telling here! 

Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |       X    |              |            |

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
+###The Early Days
+
+In the beginning, Little Wael grew up in a dirty and poor part of England called West London. In his early years, he demonstrated his potential through a number of impressive achievements, which include:
+
+- Starting fights
+- Rarely washing
+- Being profoundly disagreeable
+- Failing school
+- Getting back into school
+- Getting really good grades and stuff
```
Comment (maddiherne) [inline README.md:14]: Great use of bullet points here! Very good organization!

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+
+Because of his impressive career background and talented skill portfolio, Little Wael was offered a place at the University of Cambridge, which is way older than your whole entire country and stuff (see Figure 1 for details).
+
+![Figure 1: How Cambridge University is Literally Even Older Than Your Whole Entire Country and Stuff](https://raw.githubusercontent.com/waeltmiller/hw01-1/master/Screen%20Shot%202019-10-04%20at%2018.17.24.png)
+
+###Getting Into Science
```
Comment (maddiherne) [inline README.md:20]: I really like how you used an extra sub-header here!

File: README.md [hunk 3]
```diff
@@ -1 +1,27 @@
...
+Little Wael soon evolved into his final form as Bigger Wael (though alas, he would never grow taller than [five feet and eleven inches in height](https://i.4pcdn.org/pol/1510707466870.jpg)). Although Bigger Wael acheived top grades early on in his humanities degree,[^1] he ultimately came to the conclusion that any debate on real-world issues that eschews evidence over anecdote and is unhindered by the burden of proof is much akin to playing World of Warcraft: there seems to be a lot of action going on, but the ~~lvlcap~~ goalposts keep expanding and you never really get anywhere.
```
Comment (maddiherne) [inline README.md:21]: Interesting use of the image here for context! Imbedding the image was a cool way to incorporate this into the bio.

**Review by maddiherne — COMMENTED** [review]
* Great detail here! You really went the extra mile.
* You met all of the criteria and more.
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X    |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |      X     |              |            |
| **Ease of access**          |      X     |              |            |



You

File: README.md [hunk 4]
```diff
@@ -1 +1,27 @@
-# hw01
\ No newline at end of file
+# hw01 This is a line from RStudio
+
+##A Brief Biography of Myself, Wael Taji Miller
+
+###The Early Days
+
```
Comment (rrodriguezbarron) [inline README.md:6]: These headers don't work. To see why, review the Markdown guide.

**Review by rrodriguezbarron — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |              |     x       |
| **Coding strategy**         |           |         x     |            |
| **Presentation**            |           |         x     |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |           |     x         |            |

Remarks:

* You need to work on your headers, right now they don't work correctly. 
* Your image and link worked fine, as well as your list
*Glad to know that you coded more emphasis than just bold or italics
* Loved the Baywatch picture

File: README.md (hunk 4 above)
Comment (rrodriguezbarron) [reply README.md:6]: Hint: " "

**Pull request closed without merging**